web/channels/stream_channel.ex

```diff
@@ -21,15 +25,23 @@ defmodule Juicebox.StreamChannel do
   def handle_out("queue.updated", payload, socket) do
     IO.inspect "Queue updated"
 
-    push socket, "queue.updated", payload
+    push socket, "remote.action", %{ payload | type: "QUEUE_UPDATED" }
```

Comment:
Could we use the `QUEUE_UPDATED` on the server too, instead of `"queue.updated"`? Then the default could just be to forward blind, something like:

```elixir
def handle_out(action_type, payload, socket) do
  push socket, "remote.action", %{ payload | type: action_type }
  {:noreply, socket}
end
```

If we needed to override it then we could just intercept them here.